app/views/comments/create.js.erb

```diff
@@ -0,0 +1 @@
+$("#comments").html("<%= j render 'comments', commentable: @comment.commentable, new_comment: @new_comment %>")
```

Comment:
Personal preference, but I prefer to use data attributes, e.g. `data-behavior="comment-container" for things related to scripting in order to be able to fully leave IDs and classes to styling.